eng/common/pipelines/templates/steps/set-test-pipeline-version.yml

```diff
@@ -1,15 +1,19 @@
 parameters:
   PackageName: ''
+  PackageNames: ''
   ServiceDirectory: ''
   TestPipeline: false
 
 steps:
 - ${{ if eq(parameters.TestPipeline, 'true') }}:
     - task: PowerShell@2
       displayName: Prep template pipeline for release
-      condition: and(succeeded(), ne(variables['Skip.SetTestPipelineVersion'], 'true')) 
+      condition: and(succeeded(), ne(variables['Skip.SetTestPipelineVersion'], 'true'))
       inputs:
-        pwsh: true
         workingDirectory: $(Build.SourcesDirectory)
         filePath: $(Build.SourcesDirectory)/eng/common/scripts/SetTestPipelineVersion.ps1
-        arguments: '-BuildID $(Build.BuildId) -PackageName ${{ parameters.PackageName }} -ServiceDirectory ${{ parameters.ServiceDirectory }}'
\ No newline at end of file
+        arguments: >
+          -BuildID $(Build.BuildId)
+          -PackageNames '${{ coalesce(parameters.PackageName, parameters.PackageNames) }}'
```

Comment:
Assuming multiple template packages, yes. I wouldn't really expect multiple template service directories. Though if that were to become the case, this entire infrastructure would have to be rewritten, requiring new yml arguments, which would be much more disruptive, requiring changes to every language's yml files that use it. Since Java is currently the only consumer, due to batch release testing requiring multiple template libraries, I felt it was unnecessary at this point.